explainer/index.bs

```diff
@@ -178,6 +178,312 @@ Unlike invalidity, destroyed-ness can change after creation, is not contagious,
 
 ## Buffer Mapping ## {#buffer-mapping}
 
+A `GPUBuffer` represents a memory allocations usable by other GPU operations.
+This memory can be accessed linearly, contrary to `GPUTexture` for which the actual memory layout of sequences of texels are unknown. Think of `GPUBuffers` as the result of `gpu_malloc()`.
+
+**CPU&rarr;GPU:** When using WebGPU, applications need to transfer data from JavaScript to `GPUBuffer` very often and potentially in large quantities.
+This includes mesh data, drawing and computations parameters, ML model inputs, etc.
+That's why an efficient way to update `GPUBuffer` data is needed. `GPUQueue.writeBuffer` is reasonably efficient but includes extra copies compared to the buffer mapping used for writing buffers.
```

Comment:
Changed to "at least an extra copy". The number of extra copies depends on how optimized this path is in browsers (as in it will be more than 1 to start, but will later be 1)